command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 01:35:27PM -0700, Mike Brown wrote:
> > @@ -0,0 +1,189 @@
> > +# Operations
> > +
> > +A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
> 
> @julz Since the primary interface to customers (for a web interface)
> would not be the testing shim. I argue the customer facing interface
> is the one that needs to be tested for compliance purposes not some
> shim.

The primary interface for customers is probably never going to be
something that the OCI is certifying (it will be much further up the
stack).  The OCI just needs to certify that the runtime engine works,
and somebody else gets to figure out if the higher layers between the
runtime and the customer are functional.  I'm not opposed to having an
in-spec HTTP API in theory, but I'd rather not have an in-spec API per
runtime implementation ;).  And if we can stick to a single API in the
spec (HTTP, CLI, or otherwise) that will make testing simpler and more
obviously consistent.

> @wking you need to PR the runtime-spec, and get it approved, if you
> want to change the runtime-spec. Changing an older parent spec from
> a newer child document.. probably not a good idea.

Agreed.  Can you point out where I'm suggesting changing the semantics
of an older spec?